recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,11 @@
+MACOSX_SDK_VERSION:
```

Comment:
```suggestion
# The required version for Qt is actually 12, but 0002-avoid-using-macos-12-resource.patch removes this requirement and brings it back to v11.
# 11.3 is used to match the version for arm-64 rather than having two versions.
MACOSX_SDK_VERSION:
```